src/ReactWrapper.jsx

```diff
@@ -885,12 +885,23 @@ class ReactWrapper {
   }
 
   /**
-   * Returns true if the current wrapper has no nodes. False otherwise.
+   * Delegates to exists()
    *
    * @returns {boolean}
    */
   isEmpty() {
-    return this.length === 0;
+    // eslint-disable-next-line no-console
+    console.warn('Deprecated method isEmpty() called, use exists() instead.');
```

Comment:
@blainekasten I definitely think prefixing with `Enzyme::` makes sense, however after thinking more about it, I'm not so sure about logging it only once.

If someone uses `isEmpty()` a lot isn't it better to make noise to encourage people to make the switch sooner rather than later so the tech debts lifespan is short?

The noise was very helpful for me, after switching off mocha `--reporter dot`, to identify and locate specs using `isEmpty` so that I could update them, as `isEmpty()` is the equivalent  of `!exists()` it wasn't a matter of doing a global search and replace on the specs.